examples/angular/src/pages/ui/components/authenticator/sign-up-with-attributes/sign-up-with-attributes.component.html

```diff
@@ -1,4 +1,15 @@
 <amplify-authenticator initialState="signUp">
+  <ng-template amplifySlot="force-new-form-fields">
+    <amplify-force-new-password-form-fields></amplify-force-new-password-form-fields>
+    <amplify-text-field
+      label="Zone Info"
+      id="12233"
+      autocomplete="false"
+      placeholder="Zone Info"
+      name="zoneinfo"
+      type="text"
+    ></amplify-text-field>
+  </ng-template>
```

Comment:
Apologies for the late review. QQ, is there a zero-config value we can use?

Reply:
There are a handful of attributes that we decided that we aren't going to have default mappings for because they are app specific, including zoneinfo. As mentioned here https://ui.docs.amplify.aws/components/authenticator#signupattributes

The `zoneInfo` is in the zero config, you are right. We just don't have a mapping for it.

Reply:
Ah gotcha, I was lost yesterday xD